docs/commands/mcfunctions.md

```diff
@@ -0,0 +1,152 @@
+
+<Spoiler title="Sample 📄 manifest.json">
+
+<CodeHeader>BP/manifest.json</CodeHeader>
+
+```json
+{
+    "format_version": 2,
+    "header": {
+        "description": "Write Your Pack Description Here",
+        "name": "Write Your Pack Name Here",
+        "uuid": "00000000-0000-0000-0000-000000000000",
+        "version": [ 1, 0, 0 ],
+        "min_engine_version": [ 1, 19, 73 ]
+    },
+    "modules": [
+        {
+            "description": "§r",
+            "type": "data",
+            "uuid": "00000000-0000-0000-0000-000000000000",
+            "version": [1, 0, 0 ]
+        }
+    ]
+}
+```
+
+Note that the uuid field needs to be replaced with an actual uuid, and the two generated must be different from one another. You can generate a uuid at https://uuidgenerator.net/
+
+</Spoiler>
+<Spoiler title="Sample 🖼 pack_icon.png">
+	
+![pack_icon.png](/assets/images/commands/pack_icon.png)
+
+</Spoiler>
+
+4. Create a `📁 functions` folder. Any file within this folder that ends with **.mcfunction** will be registered as a function in-game, which can be run with `/function <function_name>`.
+    - Nested functions are allowed, simply list the file path in relation to the functions folder as shown in the function pack folder structure.
+
+5. Apply the behavior pack in-game and try out the functions. Function file changes can be reflected in the world by running `/reload` or by simply relogging.
+
+:::tip NOTE
+Functions are versioned; therefore, they will run in the version listed in the `📄 manifest.json`, such as:
+- `min_engine_version` 1.19.50 or above will adopt the new execute syntax.
+- `min_engine_version` 1.19.70 or above will require aux values be replaced with block states.
+:::
+
+## Execution
+
+Functions can be executed in-game by typing `/function name_of_function`. This will execute all the commands in the function file, all in a single tick. 
+
+Nested functions, for example `BP/functions/Lobby/items/1.mcfunction` can be run using the nested folder path, in this case `/function Lobby/items/1`
+
+## tick.json
+
+The final file within a function is the **tick.json** file. This specifies functions to run server-side on every game tick, (similar to a repeating command block.) It is located in the `BP/functions` folder. By default, functions running in this file execute at origin `0, 0, 0` in the overworld.
+
+<CodeHeader>BP/functions/tick.json</CodeHeader>
+```json
+{
+  "values": [
+    "function_1",
+    "function_2"
+  ]
+}
+```
+> Note; functions in this file are run as soon as the world is *initialized*, regardless of whether or not the player has been *loaded*. This may cause unintended behavior if used incorrectly.
+
+## Sample Function Pack
+
+<CardLink
```

Comment:
Please use github host, either in the wiki example folder or [the wiki addon repo](https://github.com/Bedrock-OSS/wiki-addon) instead of google drive